router/usecase/dynamic_splits.go

```diff
@@ -173,6 +139,7 @@ func getSplitQuote(ctx context.Context, routes []route.RouteImpl, tokenIn sdk.Co
 		isAmountInNilOrZero := inAmount.IsNil() || inAmount.IsZero()
 		isAmountOutNilOrZero := outAmount.IsNil() || outAmount.IsZero()
 		if isAmountInNilOrZero && isAmountOutNilOrZero {
+			// totalIncrementsInSplits += currentRouteIncrement
```

Comment:
I guess there is no need to keep this line anymore?

Reply:
Removed